src/main/scala/tile/Core.scala

```diff
@@ -29,6 +29,7 @@ trait CoreParams {
   val useCryptoNIST: Boolean
   val useCryptoSM: Boolean
   val useRVE: Boolean
+  val useConditionalZero: Boolean
```

Comment:
You should also add a `usingConditionalZero` in HasCoreParameters, then you can substitute all the `coreParams.useConditionalZero` to `usingConditionalZero`.

As of now, Zicond is not implemented in ABLU, so you may also add a requirement in `HasRocketCoreParameters` that `require(!(usingABLU && usingConditionalZero), "Zicond is not implemented in ABLU")` and remove such requirement when implementing that for ABLU.